Chore/update setup instructions

tahb opened the pull request:
<!-- Do you need to update the changelog? -->

## Changes in this PR

Update set up instructions to get the app running with less errors.

tahb committed:
Update example .env to get the app running first time

erbridge commented on .gitignore:
```diff
@@ -46,6 +46,7 @@
 ### Environment variables
 .env
 .env.*
+!.env.example
```
Our instructions say use for the force add for this instead:
```
git add path/to/file --force
```

tahb replied:
I was thinking this probably ought to be a default rule and in the [rails-template](https://github.com/dxw/rails-template/blob/develop/.env.example) itself since it's the one that adds and then expects the file `.env.example`?

tahb replied:
Arriving back to this a while later.

My view is still that the file called `.env.example` should be ignored by default and this change should be pushed up into [the template](https://github.com/dxw/rails-template/blob/develop/.env.example)?

erbridge replied:
I have opinions, but let's not block this with them any further.

tahb committed:
Fix setup by renaming .env.example

script/setup requires .env.example (note the first fullstop). When running this script on a clean repo, it fails. It seems like the gitignore rules prevented this file from being committed and that this problem exists in the rails-template too.

Nothing secret should ever be in the .example file.
